### PR TITLE
[MIG] account_netting: Migration to 10.0

### PR DESCRIPTION
**Pull request by cubells**
- Update Views
- Update Tests

cc @Tecnativa

### CODE REVIEW

File: account_credit_control/security/ir.model.access.csv [hunk 1]
```diff
@@ -15,6 +15,3 @@ account_credit_control.ir_model_access_286,credit_control_info_policy,account_cr
 account_credit_control.ir_model_access_287,credit_control_manager_level,account_credit_control.model_credit_control_policy_level,group_account_credit_control_manager,1,1,1,1
 account_credit_control.ir_model_access_288,credit_control_user_level,account_credit_control.model_credit_control_policy_level,group_account_credit_control_user,1,0,0,0
 account_credit_control.ir_model_access_289,credit_control_info_level,account_credit_control.model_credit_control_policy_level,group_account_credit_control_info,1,0,0,0
-account_credit_control.ir_model_access_290,credit_control_fin_user_line,account_credit_control.model_credit_control_line,account.group_account_user,1,0,0,0
-account_credit_control.ir_model_access_291,credit_control_fin_invoice_line,account_credit_control.model_credit_control_line,account.group_account_invoice,1,0,0,0
-account_credit_control.ir_model_access_292,credit_control_fin_manager_line,account_credit_control.model_credit_control_line,account.group_account_manager,1,1,1,1
```
Comment (adrienpeiffer) [inline account_credit_control/security/ir.model.access.csv]: Why this change ?

File: account_netting/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    'name': 'Account netting',
+    'version': '10.0.1.0.0',
+    'summary': "Compensate AR/AP accounts from the same partner",
+    'category': 'Accounting & Finance',
+    'author': 'Tecnativa, '
+              'Odoo Community Association (OCA)',
+    'website': 'https://tecnativa.com',
```
Comment (adrienpeiffer) [inline account_netting/__manifest__.py]: COuld you change the website according guidelines ?

**Review by adrienpeiffer — CHANGES_REQUESTED** [review]

**Review by adrienpeiffer — APPROVED** [review]
:+1: (Code review)

**Review by gurneyalex — APPROVED** [review]

File: account_netting/wizards/account_move_make_netting.py [hunk 3]
```diff
@@ -1,21 +1,33 @@
...
-    journal = fields.Many2one(
-        comodel_name="account.journal", required=True,
-        domain="[('type', '=', 'general')]")
-    move_lines = fields.Many2many(comodel_name="account.move.line")
-    balance = fields.Float(readonly=True)
+    journal_id = fields.Many2one(
+        comodel_name="account.journal",
+        required=True,
+        domain="[('type', '=', 'general')]",
+        oldname='journal',
```
Comment (pedrobaeza) [inline account_netting/wizards/account_move_make_netting.py]: Not needed. It's a transient model

File: account_netting/tests/test_account_netting.py [hunk 4]
```diff
@@ -1,77 +1,134 @@
...
-        workflow.trg_validate(
-            self.uid, 'account.invoice', self.supplier_invoice.id,
-            'invoice_open', self.cr)
-        supplier_move = self.supplier_invoice.move_id
-        self.move_line_2 = supplier_move.line_id.filtered(
-            lambda x: x.account_id == self.account_payable)
+        res_users_account_manager = cls.env.ref(
+            'account.group_account_manager')
+        partner_manager = cls.env.ref('base.group_partner_manager')
+        cls.account_manager = cls.env['res.users'].with_context(
```
Comment (pedrobaeza) [inline account_netting/tests/test_account_netting.py]: Don't create a new user, but just add the corresponding group to your user (KISS principle)

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
And review the rest of the comments

File: account_netting/tests/test_account_netting.py (hunk 4 above)
Comment (pedrobaeza) [reply account_netting/tests/test_account_netting.py]: This hasn't been attended

Comment (cubells) [reply account_netting/tests/test_account_netting.py]: You are right

File: account_netting/tests/test_account_netting.py [hunk 5]
```diff
@@ -18,20 +18,14 @@ def setUpClass(cls):
...
-        ).create({
-            'name': 'Adviser',
-            'company_id': cls.env.user.company_id.id,
-            'login': 'fm',
-            'email': 'accountmanager@yourcompany.com',
+        cls.env.user.write({
             'groups_id': [
                 (6, 0, [res_users_account_manager.id, partner_manager.id])
-            ],
+            ]
```
Comment (pedrobaeza) [inline account_netting/tests/test_account_netting.py]: Don't remove comma

File: account_netting/tests/test_account_netting.py [hunk 6]
```diff
@@ -18,20 +18,14 @@ def setUpClass(cls):
...
         })
         # only adviser can create an account
         cls.account_receivable = cls.env['account.account'].sudo(
-            cls.account_manager.id
+            cls.env.user.id
```
Comment (pedrobaeza) [inline account_netting/tests/test_account_netting.py]: No need to make sudo now

File: account_netting/tests/test_account_netting.py [hunk 7]
```diff
@@ -40,7 +34,7 @@ def setUpClass(cls):
             'reconcile': True,
         })
         cls.account_payable = cls.env['account.account'].sudo(
-            cls.account_manager.id
+            cls.env.user.id
```
Comment (pedrobaeza) [inline account_netting/tests/test_account_netting.py]: No need to make sudo now

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by pedrobaeza — APPROVED** [review]
